### PR TITLE
feat(onboarding): Add warning for when docs are missing examples

### PR DESCRIPTION
**Pull request by evanpurkhiser**
We mark the docs that are missing 'how to send verification event' sections in https://github.com/getsentry/sentry-docs/pull/1034

This displays an warning alert at the bottom of the docs (for when the user can't figure out how to send their first event).

/cc @MimiDumpling for wording
/cc @eurica / @benvinegar for any thoughts as well

I'm def open to the wording implying a bit less like we haven't written them.. But I just want users to not be frustrated when we tell them our app is waiting to receive their first event, but then we don't even tell them how to send it.

image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/onboarding/projectSetup/projectDocs.jsx [hunk 1]
```diff
@@ -249,6 +285,13 @@ const DocsWrapper = styled(posed.div(docsTransition))`
     margin-bottom: 1rem;
   }
 
+  /**
+   * XXX(epurkhiser): This comes from the doc styles and avoids bottom margin issues in alerts
+   */
+  .content-flush-bottom *:last-child {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/onboarding/projectSetup/projectDocs.jsx]: Can we style the alert instead of relying on `last-child`?

**Review by billyvg — APPROVED** [review]
How are TODOs handled? do we track/measure how often these projects are hit so know which examples to prioritize?

File: src/sentry/static/sentry/app/views/onboarding/projectSetup/projectDocs.jsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/onboarding/projectSetup/projectDocs.jsx]: No these classes come from the html generated by `docs.sentry.io`'s API.

**Comment by EvanPurkhiser** [comment]
> do we track/measure how often these projects are hit so know which examples to prioritize?

Yep, this is actually how the projects that didn't get updated were selected.

File: src/sentry/static/sentry/app/views/onboarding/projectSetup/projectDocs.jsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/onboarding/projectSetup/projectDocs.jsx]: ah actually this should be in a different PR

**Pull request merged**